components/projectPage/languages/Languages.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import style from './Languages.module.scss';
+
+function Languages() {
+  const languages = ['HTML', 'CSS', 'JavaScript']; //in the future: API
+  const languagesArr = languages.map((language) => {
+    return (
+      <li key={language} className={style.languages}>
```

Comment:
why did you put language in the key?

Reply:
because I assumed we are not going to write the same language twice, so it can be good as a key-value, should I change it to index number?